### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by hjmodi**

### CODE REVIEW

File: src/main/resources/META-INF/resources/js/keycloak.js
Comment (ligangty) [inline src/main/resources/META-INF/resources/js/keycloak.js:1]: Why do we need this file?

**Review by ligangty — COMMENTED** [review]
One little issue.

File: src/main/resources/META-INF/resources/js/keycloak.js
Comment (hjmodi) [reply src/main/resources/META-INF/resources/js/keycloak.js:1]: hmm, right. Removed it.

**Review by ligangty — APPROVED** [review]
LGTM

**Pull request merged**